src/touchid/TouchID.mm

```diff
@@ -95,18 +95,24 @@ inline QString hash(const QString& value)
     // prepare adding secure entry to the macOS KeyChain
     CFErrorRef error = NULL;
     SecAccessControlRef sacObject;
+#if __clang_major__ >= 9
     if (@available(macOS 10.15, *)) {
+#if MAC_OS_X_VERSION_MIN_REQUIRED >= 101500
```

Comment:
Can you combine these as well like the others? I like that presentation better and the effect is the same at the end of the day.

Reply:
Done.